style.css

```diff
@@ -0,0 +1,1491 @@
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 20px;
+    text-align: center;
+    color: white;
+    display: flex;
+  }
+
+  .buttons-up {
+    float: right;
+    margin-right: 156px;
+    margin-top: -20px;
+  }
+
+  .buttons-up ul {
+    list-style: none;
+    display: flex;
+    text-decoration: none;
+    color: white;
+    gap: 32px;
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .buttons-up ul li a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .headline {
+    display: none;
+  }
+
+  .alex {
+    width: 100%;
+    height: 160px;
+    padding-left: 143px;
+    padding-top: 264px;
+    font-family: "Crete Round", serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    text-align: left;
+    color: #f55800;
+  }
+
+  .soft {
+    font-family: "Inter", serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    text-align: left;
+    color: #fff;
+    padding-top: 170px;
+    padding-left: 143px;
+    width: 100%;
+    height: 24px;
+  }
+
+  .text {
+    font-family: "Inter", serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    text-align: left;
+    color: #f4f5f7;
+    width: 700px;
+    height: 566px;
+    padding-left: 143px;
+    padding-top: 60px;
+    max-width: fit-content;
+  }
+
+  .grid-headline {
+    display: grid;
+    grid-template-columns: (2, 1 fr);
+    background-color: #1c1a19;
+    height: 870px;
+  }
+
+  .headline2 {
+    background-image: url("icons/desktop-ill1.png");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 45%;
+    background-position-y: 5px;
+  }
+
+  .icon-desk {
+    list-style: none;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    margin-top: -650px;
+    padding-left: 18px;
+    gap: 15px;
+  }
+
+  #work {
+    width: 100%;
+    display: grid;
+    max-width: fit-content;
+  }
+
+  .recent-works {
+    font-family: "Crete Round", serif;
+    font-size: 40px;
+    font-weight: 400;
+    line-height: 52px;
+    text-align: left;
+    color: #091e42;
+    padding-left: 153px;
+    padding-top: 135px;
+    width: 100%;
+  }
+
+  .vector {
+    display: none;
+  }
+
+  .gridContainer {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 20px;
+    height: 448px;
+    margin: 60px 50px 0 70px;
+  }
+
+  .container2 {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 15px;
+    height: 907px;
+    padding: 20px;
+    margin-bottom: 120px;
+    padding-left: 4%;
+    width: 100%;
+  }
+
+  .background1 {
+    background-image: url("icons/desk1.png");
+    background-repeat: no-repeat;
+    width: 100%;
+    background-size: cover;
+    height: 435px;
+    padding: auto;
+    margin-top: 20px;
+  }
+
+  .background1:hover {
+    border: 1px solid black;
+  }
```

Comment:
- [ ] Ouch! this is really close but as mentioned by the previous reviewer when you hover over the project card the "See project" button should be displayed. 
- [ ] This means you need to add a "See project" button to each project card with an initial `display: none;` property and when the card is hovered over you can display the button using `display: block;` 😉
    - Keep in mind the above is just an example to explain the underlying concept you can do this in other ways.